recipes/raylib/all/conanfile.py

```diff
@@ -22,11 +23,21 @@ class RaylibConan(ConanFile):
         "shared": [True, False],
         "fPIC": [True, False],
         "opengl_version": [None, "4.3", "3.3", "2.1", "1.1", "ES-2.0"],
+
+        "module_raudio": [True, False],
+
+        "events_waiting": [True, False],
+        "custom_frame_control": [True, False]
     }
     default_options = {
         "shared": False,
         "fPIC": True,
         "opengl_version": None,
+
+        "module_raudio": True,
+
+        "events_waiting": False,
+        "custom_frame_control": False
```

Comment:
```suggestion
        "customize_build": False,
        "module_raudio": True,
        "events_waiting": False,
        "custom_frame_control": False
```

Follow the upstream: https://github.com/raysan5/raylib/blob/5.0/CMakeOptions.txt#L11